powerline/segments/i3wm.py

```diff
@@ -28,7 +29,8 @@ def workspaces(pl, only_show=None, output=None, strip=0):
 		are shown.
 
 	:param str output:
-		If specified, only workspaces on this output are shown.
+		If specified, only workspaces on this output are shown. If unspecified, 
+		may be set in ``segment_info``.
```

Comment:
If you check documentation on readthedocs you will see that segment documentation never contains `pl` or `segment_info` in the argument list. It is removed by Powerline mocked version of autodoc, intentionally.